elastic/kibana
# [SecuritySolution] Fix entity-store to support asset criticality delete

## Summary

Update the entity store API so it does not return the asset criticality field when the value is 'deleted'.


### How to test it
* Open kibana with data
* Install the entity store
* Update asset criticality for a host or user
* Wait for the engine to run (I don't know a reliable way to do this)
* Refresh the entity analytics dashboard, and it should show empty fields for deleted asset criticality

- [ ] Backport it to 8.16

### Checklist
